src/main/web_src/src/pages/testnorgePage/TestnorgePage.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react'
+import Title from '~/components/Title'
+import { Formik } from 'formik'
+import SearchContainer from '~/components/SearchContainer'
+import SearchOptions from './SearchOptions'
+import PersonSearch from '~/service/services/personsearch'
+import SearchView from '~/pages/testnorgePage/SearchView'
+import _get from 'lodash'
+import { Person } from '~/service/services/personsearch/types'
+
+export default () => {
+	const [items, setItems] = useState<Person[]>([])
+	const [page, setPage] = useState(1)
+	const [pageSize] = useState(20)
+	const [numberOfItems, setNumberOfItems] = useState<number | null>(null)
+
+	const search = (page: number, values: any) =>
+		PersonSearch.search({
+			pageing: {
+				page: page,
+				pageSize: pageSize
+			},
+			kjoenn: values?.personinformasjon?.diverse?.kjoenn,
+			foedsel: {
+				fom: values?.personinformasjon?.alder?.foedselsdato?.fom,
+				tom: values?.personinformasjon?.alder?.foedselsdato?.tom
+			},
+			statsborgerskap: {
+				land: values?.personinformasjon?.statsborgerskap?.land
+			},
+			sivilstand: {
+				type: values?.personinformasjon?.sivilstand?.type
+			},
+			tag: 'TESTNORGE'
+		}).then(response => {
+			setPage(page)
+			setItems(response.items)
+			setNumberOfItems(response.numerOfItems)
+		})
+
+	useEffect(() => {
+		search(1, null)
+	}, [])
+
+	const onSubmit = (values: any) => search(1, values)
+
+	return (
+		<div>
+			<Title title="Søk i Testnorge" beta={true} />
+			<p>Testnorge er en felles offentlig testdata populasjon.</p>
```

Comment:
Rett til "testdatapopulasjon".
Og kunne vi hatt denne teksten i en infoboks (som vi har på mini-norge)? Synes den blir litt malplassert når den bare står der for seg selv 🤔

Reply:
Altså grunnen til at jeg tokk den med der fordi vi har en oversikt også har vi forklaringen under. Som man er vant med på andre nettsider. Tror gruppen til at den er malpalert er at det litt for lite tektst?

Reply:
Testnorge er en felles offentlig testdatapopulasjon, som ble laget i forbindelse med nytt folkeregister. Populasjonen er levende, og endrer seg fortløpende ved at personer fødes, dør, får barn, osv. Pr. i dag støttes kun personer, men fremover vil det også komme støtte for organisasjoner og arbeidsforhold. 

Testnorge er tilgjengelig i PDL.